documentation/tutorial/07-deploy-ingress.md

```diff
@@ -0,0 +1,183 @@
+nginx-ingress-integrator/0*  active    idle   10.1.232.73          Ingress IP(s): 127.0.0.1, 127.0.0.1, Service IP(s): 10.152.183.172, 10.152.183.235
+postgresql-k8s/0*            active    idle   10.1.232.66          Primary
+temporal-admin-k8s/0*        active    idle   10.1.232.71
+temporal-k8s/0*              active    idle   10.1.232.64
+temporal-ui-k8s/0*           active    idle   10.1.232.72
+
+```
+
+## Verify Ingress Resource
+
+To verify the ingress resources were correctly created, you can run the
+following command:
+
+```bash
+kubectl describe ingress -n temporal-model
+```
+
+The output should look similar to the following (with the exception of the
+service IP addresses):
+
+```
+Name:             temporal-k8s-ingress
+Labels:           app.juju.is/created-by=nginx-ingress-integrator
+                  nginx-ingress-integrator.charm.juju.is/managed-by=nginx-ingress-integrator
+Namespace:        temporal-model
+Address:          127.0.0.1
+Ingress Class:    public
+Default backend:  <default>
+TLS:
+  temporal-tls terminates temporal-k8s
+Rules:
+  Host          Path  Backends
+  ----          ----  --------
+  temporal-k8s
+                /   temporal-k8s-service:7233 (10.1.232.64:7233)
+Annotations:    nginx.ingress.kubernetes.io/backend-protocol: GRPC
+                nginx.ingress.kubernetes.io/proxy-body-size: 20m
+                nginx.ingress.kubernetes.io/proxy-read-timeout: 60
+                nginx.ingress.kubernetes.io/rewrite-target: /
+Events:         <none>
+
+
+Name:             temporal-ui-k8s-ingress
+Labels:           app.juju.is/created-by=nginx-ingress-integrator
+                  nginx-ingress-integrator.charm.juju.is/managed-by=nginx-ingress-integrator
+Namespace:        temporal-model
+Address:          127.0.0.1
+Ingress Class:    public
+Default backend:  <default>
+TLS:
+  temporal-tls terminates temporal-ui-k8s
+Rules:
+  Host             Path  Backends
+  ----             ----  --------
+  temporal-ui-k8s
+                   /   temporal-ui-k8s-service:8080 (10.1.232.72:8080)
+Annotations:       nginx.ingress.kubernetes.io/backend-protocol: HTTP
+                   nginx.ingress.kubernetes.io/proxy-body-size: 20m
+                   nginx.ingress.kubernetes.io/proxy-read-timeout: 60
+                   nginx.ingress.kubernetes.io/rewrite-target: /
+Events:            <none>
+```
+
+## Connect Ingress
+
+Once deployed and related, find the IP of the ingress controller by running the
+following command:
+
+```bash
+kubectl get pods -n ingress -o wide
+```
+
+You should see the following output:
+
+```
+NAME                                      READY   STATUS    RESTARTS          AGE    IP           NODE      NOMINATED NODE   READINESS GATES
+nginx-ingress-microk8s-controller-mfmtx   1/1     Running   512 (3h15m ago)   145d   10.1.232.8   ubuntu   <none>           <none>
+```
+
+and add the IP-to-hostname mapping in your /etc/hosts file as follows:
```

Comment:
I think it would be better to quote `/etc/hosts` with backticks to separate it from the text.